src/main/rules/GCI1024/IaC/kubernetes/GCI1024.asciidoc

```diff
@@ -0,0 +1,54 @@
+Chaque conteneur défini dans un Pod, un Deployment, ou un StatefulSet Kubernetes doit obligatoirement déclarer une readinessProbe et une livenessProbe.
```

Comment:
please, translate the description in english because our plugins are international